xds/internal/resolver/serviceconfig.go

```diff
@@ -121,6 +121,7 @@ type routeCluster struct {
 
 type route struct {
 	m                 *xdsresource.CompositeMatcher // converted from route matchers
+	actionType        xdsresource.RouteActionType   //holds route action type from type_rds.Route
```

Comment:
```suggestion
	actionType        xdsresource.RouteActionType   //holds route action type 
```

Reply:
Nit: also please add a space in between // and comment.